lib/index.js

```diff
@@ -31,29 +40,21 @@ function NgFaker(options) {
         return obj;
     }
 
-    var Fake = require('./fake');
     this.fake = new Fake(self).fake;
 
-    var Helpers = require('./helpers');
     this.helpers = new Helpers(self);
 
-    var Name = require('./name');
     self.name = bindAll(new Name(self));
```

Comment:
Is this what you're talking about @andela-rbabalola ?

Reply:
I see 🙈 I was confused there